### ISSUE TITLE
parse error

### ISSUE THREAD

**Issue by matthewberryman** [MSFREACH/msf-reach#381]
https://github.com/MSFREACH/msf-reach/blob/4fc44dce8a667e1a696c62680a58da12e1fd4f1c/src/api/routes/contacts/model.js#L158

This line's throwing a parse error when going through the API but when I copy and paste it and put in the actual values (below), it's fine. Is node-pg adding quote marks or something around `$1`?

Note: use latest schema in dev branch of schema repo.

```sql
update cognicity.contacts set properties = jsonb_set(properties::jsonb,'{"sharedWith"}', ((properties->'sharedWith')::jsonb || '["f5096c9c-ed52-4deb-bc23-2530287a5daf"]'::jsonb)) where id=1;
```

tag #332

**Comment by mehrdadgit** [comment]
@matthewberryman you had already fixed this I think; I just did a bit of tidy up; It should work but please test and revert if not

**Comment by matthewberryman** [comment]
@mehrdadgit nope it broke things. Please focus on open issues (noting this one I had written a commit with "fix" in the message, just wasn't closed as I haven't merged back to master as I'm still working on #332 )

**Comment by mehrdadgit** [comment]
Ok sure, what did it break by the way? (just for reference)  cause travis build fails regardless...
